EIPS/eip-3722.md

```diff
@@ -0,0 +1,164 @@
+---
+eip: 3722
+title: Poster
+author: Auryn Macmillan (@auryn-macmillan)
+discussions-to: (https://ethereum-magicians.org/t/eip-poster-a-ridiculously-simple-general-purpose-social-media-smart-contract/6751)
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-31
+---
+
+# [Poster](https://github.com/ETHPoster/contract)
```

Comment:
```suggestion
# [Poster](https://github.com/ETHPoster/contract)
```
This is not a valid EIP header.

Reply:
Because it is a link? Or for some other reason?